packages/client/test/unit/proxy.test.js

```diff
@@ -0,0 +1,80 @@
+import { ProxyHttpAgent, ProxyHttpsAgent, createPacAgent, getProxy, proxyAgentFor } from '../../src/proxy.js';
+import { PacProxyAgent } from 'pac-proxy-agent';
+
+describe('proxy', () => {
+
+  describe('getProxy', () => {
+    it('should return proxy object if proxy is set', () => {
+      process.env.http_proxy = 'http://proxy.com:8080';
```

Comment:
update env vars in before and after, so that its not skipped in case test fails.

Reply:
Changed